src/NuGet.Core/NuGet.PackageManagement/IDE/PackageRestoreManager.cs

```diff
@@ -57,6 +58,40 @@ public virtual async Task RaisePackagesMissingEventForSolutionAsync(string solut
             PackagesMissingStatusChanged?.Invoke(this, new PackagesMissingStatusEventArgs(missing));
         }
 
+        public virtual async Task<bool> GetMissingAssetsFileStatusAsync()
+        {
+            foreach (var nuGetProject in (await SolutionManager.GetNuGetProjectsAsync()))
+            {
+                if (nuGetProject is BuildIntegratedNuGetProject)
```

Comment:
```suggestion
                if (nuGetProject is BuildIntegratedNuGetProject buildIntegratedNuGetProject)
```

Using this syntax you can assign it a variable when checking if nuGetProject is a BuildIntegratedNuGetProject  and then you don't have to cast it again on line 67, you can use the variable buildIntegratedNuGetProject.